pom.xml

```diff
@@ -169,6 +170,12 @@
                 </executions>
                 <configuration>
                     <mainClass>org.keycloak.webbuilder.WebBuilder</mainClass>
+                    <systemProperties>
+                        <systemProperty>
+                            <key>version.keycloak.client</key>
+                            <value>${version.keycloak.client}</value>
+                        </systemProperty>
+                    </systemProperties>
```

Comment:
This change is not needed - you can simply do `mvn install -Dversion.keycloak.client=whatever` and it'll override the value specified in the pom.xml.

```suggestion
```